format/private/mirror_linguist_languages.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# Generate a snippet of bash code to help us detect languages based on filenames
+raw=$(mktemp --suffix=.yml)
+json=$(mktemp --suffix=.json)
+wget -O "$raw" https://raw.githubusercontent.com/github-linguist/linguist/master/lib/linguist/languages.yml
+yq -o=json '.' "$raw" > "$json"
```

Comment:
I'm sure YQ could do this in one step, but I'm already familiar with JQ and it wasn't the right day to learn a new tool.

Reply:
we don't need yq anymore. we can continue using jq

Reply:
the input is a yaml file though, so I think I do need this conversion?

Obviously this code will barely ever be run, and one could just look at the github-linguist repo and see a short list of extensions for the next language added. Maybe it's overkill to check this in at all?

Reply:
oh missed that the extension was yaml.